WcaOnRails/app/models/competition_event.rb

```diff
@@ -21,4 +21,18 @@ def to_wcif
       "rounds" => self.rounds.map(&:to_wcif),
     }
   end
+
+  def load_wcif!(wcif)
+    self.rounds.each(&:destroy!)
```

Comment:
I decided to monkeypatch `ActiveRecord::Associations::CollectionProxy` to add a `destroy_all!` method that does what I want. Hopefully that makes both of us happy =) cc-ing @jonatanklosko who wanted to use `destroy_all` in a past PR and I didn't let him.